tests/containers/realtime.pm

```diff
@@ -0,0 +1,84 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman
+# Summary: Test RT workload in a container
+# Maintainer: qa-c@suse.de
+
+use Mojo::Base qw(containers::basetest);
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils qw(script_retry);
+
+# run a RT process, change priority and scheduling policy FIFO and later change to RoundRobin
+sub test_schedule {
+    my ($is_rt, $runtime, $container) = @_;
+
+    assert_script_run("$runtime exec $container /bin/bash -c 'chrt -m'");
+    validate_script_output("$runtime exec $container /bin/bash -c 'cat /sys/kernel/realtime'", qr/^1$/);
```

Comment:
No need for `/bin/bash -c` in this command and the rest:

```suggestion
    validate_script_output("$runtime exec $container cat /sys/kernel/realtime", qr/^1$/);
```

Reply:
True, updated